recipes-bsp/imx-vpu-hantro-vc/imx-vpu-hantro-vc_1.3.0.bb

```diff
@@ -13,4 +13,29 @@ S = "${WORKDIR}/${BPN}-${PV}"
 SRC_URI[md5sum] = "37c61718602ff08041b81319cfd6a8d9"
 SRC_URI[sha256sum] = "8425796fc63d7107f2f57c51497281c4d54c883b2670817347fbe55f39dc2b8c"
 
+# For native apps, insert the user-local sysroot path
+# For nativesdk apps, insert the correct distro folder
+D_SUBDIR                 = ""
+D_SUBDIR_class-native    = "${STAGING_DIR_NATIVE}"
+D_SUBDIR_class-nativesdk = "/opt/${DISTRO}"
```

Comment:
This is handled behind the scenes by the building system. What error are you receiving without those overrides?

Reply:
ah I see. I deleted that part and left do_install.